src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeApprovedConsumer.cs

```diff
@@ -0,0 +1,97 @@
+﻿using MassTransit;
+using TrafficCourts.Messaging.MessageContracts;
+using TrafficCourts.Workflow.Service.Services;
+using TrafficCourts.Workflow.Service.Models;
+
+namespace TrafficCourts.Workflow.Service.Consumers
+{
+    /// <summary>
+    ///     Consumer for DisputeApproved message.
+    /// </summary>
+    public class DisputeApprovedConsumer : IConsumer<DisputeApproved>
+    {
+        private readonly ILogger<DisputeApprovedConsumer> _logger;
+        private readonly ISubmitDisputeToArcService _submitDisputeToArcService;
+
+        public DisputeApprovedConsumer(ILogger<DisputeApprovedConsumer> logger, ISubmitDisputeToArcService submitDisputeToArcService)
+        {
+            _logger = logger;
+            _submitDisputeToArcService = submitDisputeToArcService;
+        }
+        public async Task Consume(ConsumeContext<DisputeApproved> context)
+        {
+            try
+            {
+                if (context.RequestId != null)
+                {
+                    _logger.LogInformation("DisputeApprovedConsumer is consuming message: " + context.Message.Id);
```

Comment:
Do not use string concatenation. Use proper property name. However, this kind of log message is not really required, especially at Information level. OpenTelemetry integration with MassTransit should be providing us this tracability.

```
_logger.LogInformation("DisputeApprovedConsumer is consuming message: {MessageId}", context.Message.Id);
```